### PR TITLE
Move DesktopUI project from Runtime to WindowsUx

### PR DESCRIPTION
**Pull request by jalkire**
#### Describe the change
Thanks to the hard work of our axe-windows feature team, the DesktopUI project no longer makes much sense in the Runtime folder of our solution. This PR moves it under WindowsUx. There is no code change. I have some more ideas in mind with regard to possibly depracating this project, but this seemed like a non-controversial first-step PR to me.

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: src/AccessibilityInsights.sln [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.27130.2024
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28803.156
```
Comment (DaveTryon) [inline src/AccessibilityInsights.sln]: >16 [](start = 22, length = 2)

Is this VS2019?

Comment (jalkire) [reply src/AccessibilityInsights.sln]: Oops, yes.

**Comment by karanbirsingh** [comment]
Can you double-check the docs and change them if it makes sense now (vs the later cleanup after axe-windows)?

**Comment by jalkire** [comment]
> Can you double-check the docs and change them if it makes sense now (vs the later cleanup after axe-windows)?

It looks to me like the docs make sense as is ith regart to this change. Did you have anything in particular in mind?

**Pull request merged**